config/fortify-client.properties

```
@@ -0,0 +1 @@
+fortify.client.releaseId=134905
```

I think the release ID for frontend is 134904 (different to case-api)
```suggestion
fortify.client.releaseId=134904
```